<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>913</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRefresh"/>
   <addaction name="separator"/>
   <addaction name="actionPrevious"/>
   <addaction name="actionPlay"/>
   <addaction name="actionNext"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionPlay">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/play.png</normaloff>:/icons/play.png</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/refresh.png</normaloff>:/icons/refresh.png</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/left-arrow.png</normaloff>:/icons/left-arrow.png</iconset>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/right-arrow.png</normaloff>:/icons/right-arrow.png</iconset>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
